src/main/java/com/bookstore/model/Book.java

```diff
@@ -30,6 +30,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
-    @Column(nullable = false)
+    @Column(nullable = false, name = "is_deleted")
```

Comment:
Actually, I'm not sure if we need to explicitly add the column name here, because we do it anyway in the liquibase changes file.